src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const allPopulation = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const result = [...allPopulation].map(
```

Comment:
Instead of `result`, it's better to use something more descriptive like `totalPopulationCount`